### ISSUE TITLE
Navigation bar hides headings via anchors

### ISSUE THREAD

**Issue by justinvp** [pulumi/docs#2488]
We originally had this tracked in #1709, which @cnunciato fixed with #1771 and #1772.

However, it looks like we've regressed this recently. I'm guessing this was due to the CSS selectors being removed by PurgeCSS.

**Comment by cnunciato** [comment]
I wonder if there’s a way to run PurgeCSS without actually applying changes, like in an advisory mode? That might work out better for us actually. cc @zchase

**Comment by justinvp** [comment]
Another thought: I wonder if the `/* purgecss ignore */` [comments](https://purgecss.com/whitelisting.html#in-the-css-directly) would work in our `*.scss` files. If so, we could add `/* purgecss start ignore */` and `/* purgecss end ignore */` to most (all?) of _our_  `*.scss` files to keep the selectors we know we are using. Instead of having to keep track of them in the whitelists in `postcss.config.js`.

**Comment by zchase** [comment]
@justinvp I took a look into the approach you mentioned above & while it does work it does force us to include a ton of CSS. Before implementing PurgeCSS, we were not using around ~97% of the CSS file we were loading. With the original approach (whitelisting) we got that down to around ~85% of the bundle not used, still not great but better. By just wrapping our custom `*.scss` files it gives us ~95% of the bundle not used. So not as performant as maintaining the whitelist ourselves.

This specific issue (anchor overlap) is solved with this PR: https://github.com/pulumi/docs/pull/2489
